internal/services/servicebus/servicebus_queue_data_source.go

```diff
@@ -78,20 +58,17 @@ func dataSourceServiceBusQueue() *pluginsdk.Resource {
 				Computed: true,
 			},
 
-			// TODO 4.0: change this from enable_* to *_enabled
-			"enable_batched_operations": {
+			"batched_operations_enabled": {
```

Comment:
The way this is written is a breaking change and will need to be wrapped in 5.0 logic before renaming